### PR TITLE
Fix sector times on Qualifying Best & Last screens

### PR DESCRIPTION
**Pull request by NoComment1105**
Also does updates to dependencies and code

### CODE REVIEW

**Comment by NoComment1105** [comment]
I do indeed use IntelliJ, I'll add it to the .gitignore. IntelliJ informed me there were many vulnerabilities in the old dependencies, so I thought a general update would be ideal. Encoding go brrr, I'll sort it out.
Also when I used the Start.bat file it wouldn't properly open the application, I might be being dense but if you can reproduce that behavior it would be great

**Comment by NoComment1105** [comment]
Sorted those things out for you 👍🏼

File: changelog/ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# v1.5.6
+
+Bug Fixes:
+* Fixed a bug where third sector times will be identical to First Sector times for all cars
```
Comment (LennysLounge) [inline changelog/ChangeLog.md:4]: You can add (fixed by <your name>) at the end if you like.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```
Comment (LennysLounge) [inline gradle/wrapper/gradle-wrapper.properties:3]: Don't update gradle. Please revert.

File: base/build.gradle [hunk 3]
```diff
@@ -1,98 +1,104 @@
-plugins{
+plugins {
```
Comment (LennysLounge) [inline base/build.gradle:1]: Please also revert changes to this file

**Review by LennysLounge — COMMENTED** [review]
Dependency updates only if really necessary and in a separate PR. Its not that im fundamentally against dependecy updates its just that i would like to do them myself so i have a better understanding of what was updated and what those updates mean for the project.
Especially after the backdoor was found in the xz utils project im really hesitant to merge a random gradle_wrapper.jar file into the repo. 

You can use the "files changed" tab in the PR to look over your changes again and make sure they are actually minimal. For the encoding changes it might be good enough to go into your git client and reset those lines individually.

**Comment by LennysLounge** [comment]
@NoComment1105 , With the new Nürburgring update i made some fixed and included your fix in there to.
I thought i would be easier if i copy the necessary changed directly so we dont have to go back and forth. I also mentioned your github handle in the release notes.

**Pull request closed without merging**